editor/editor_properties.cpp

```diff
@@ -3281,6 +3281,8 @@ void EditorPropertyResource::update_property() {
 				sub_inspector->set_read_only(is_read_only());
 				sub_inspector->set_use_folding(is_using_folding());
 
+				sub_inspector->set_draw_focus_border(false);
```

Comment:
`sub_inspector` needs to grab focus, but not show the style? It seems that `sub_inspector` does not need to grab focus.